etc/scripts/dependency-check.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
```

Comment:
Could we use a different name? `dependency` sounds like general dependency checking (such as to make sure we do not have conflicting versions). Could this be `security-check.sh` or maybe even `owasp-check.sh`?

Reply:
Fixed. Now `owasp-dependency-check`